src/pages/endpoint/index.js

```diff
@@ -0,0 +1,200 @@
+    text-align: center;
+    height: 100vh;
+    padding: auto 0;
+    justify-content: start;
+`
+
+const ButtonContainer = styled.div`
+    margin-top: 2rem;
+`
+
+const InputGroup = styled.div`
+    width: 40rem;
+    margin: 0 auto 3.4rem;
+`
+
+const StyledButton = styled(Button)`
+    margin: 0.8rem 0.4rem;
+`
+
+const endpointValidation = (values) => {
+    let errors = {}
+
+    const server_url = trimSpaces(values ? values.server_url : '')
+    const app_id = trimSpaces(values ? values.app_id.toString() : '')
+    const server_url_error = validation.text(server_url)
+    const app_id_error = validation.text(app_id)
+
+    if (server_url_error) {
+        errors.server_url = server_url_error
+    }
+
+    if (app_id_error) {
+        errors.app_id = app_id_error
+    }
+
+    return errors
+}
+
+const Endpoint = () => {
+    const [server_url, setServerUrl] = useLocalStorageState(default_server_url, 'config.server_url')
+    const [app_id, setAppId] = useLocalStorageState(getAppId(), 'config.app_id')
+    const { website_status, setWebsiteStatus, website_status_loading } = React.useContext(
+        DerivStore,
+    )
+    const TIMOUT_DELAY = 1500
+
+    const handleStatus = (setStatus, message) => {
+        setStatus({ message })
+        setTimeout(() => {
+            setStatus({})
+        }, TIMOUT_DELAY)
+    }
+    const resetEndpointSettings = (setStatus) => {
+        setServerUrl(default_server_url)
+        setAppId(getAppId())
+        setWebsiteStatus()
+        handleStatus(setStatus, 'Config has been reset successfully')
+        // TODO: if there is a change requires reload in the future
+        // window.location.reload()
+    }
+    const endpointSubmission = (values, actions) => {
+        actions.setSubmitting(true)
+        setServerUrl(values.server_url)
+        setAppId(values.app_id)
+
+        // handle website status changes
+        const new_website_status = { ...website_status, clients_country: values.clients_country }
+        setWebsiteStatus(new_website_status)
+        actions.setSubmitting(false)
+        handleStatus(actions.setStatus, 'Config has been updated')
+        // TODO: if there is a change requires reload in the future
+        // window.location.reload()
+    }
+
+    return (
+        <Layout type="static" margin_top={'0'}>
+            <SEO title="Endpoint" description="Change deriv API endpoint." no_index />
+            <StyledContainer justify="center" align="center" direction="column">
+                <Header as="h2" type="page-title" align="center" mt="80px">
+                    Change API endpoint
+                </Header>
+                <Header
+                    as="h4"
+                    type="sub-section-title"
+                    align="center"
+                    weight="500"
+                    mt="0.5rem"
+                    mb="3.8rem"
+                >
+                    Update configuration for API endpoint or other settings
```

Comment:
Should this be localized?

Reply:
Don't think so as this page is only for us 😂